internal/fs/fs_test.go

```diff
@@ -21,25 +22,35 @@ func TestHasFilepathPrefix(t *testing.T) {
 	}
 	defer os.RemoveAll(dir)
 
+	// dir2 is the same as dir but with different capitalization on Windows to
+	// test case insensitivity
```

Comment:
I could have elaborated more in my comment. I did a bit of research and the information is not very thorough. The best explanation of the situation is in the answers [here](https://superuser.com/questions/266110/how-do-you-make-windows-7-fully-case-sensitive-with-respect-to-the-filesystem).
How I read this and other material I have seen is that while NTFS itself is case-sensitive, Windows is not. I.e. you can create a file with specific capitalization, but Windows will not let you create another one with different capitalization. 

Now, your point is also well made. The reason I decided to assume case-insensitivity is because I see some value in it as-is, and on the other hand I am not sure how I would write this test properly as I we don't know what file-system temp directory is on.